ng/src/web/pages/schedules/dialog.js

```diff
@@ -79,13 +76,14 @@ const ScheduleDialog = ({
 }) => {
 
   const data = {
-    ...DEFAULTS,
     ...schedule,
```

Comment:
Are there any properties beside id used from the schedule not already passed via props to ScheduleDialog? If not maybe we can avoid passing the schedule to the ScheduleDialog at all by adding id to the props.

Reply:
It seems to be the id only, so I deleted passing the whole schedule and replaced it with just the id.